refractor.js

```diff
@@ -0,0 +1,9 @@
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+    this.id = new Date().getTime().toString().concat(performance.now());
+  }
+}
+
+export default Book;
```

Comment:
- [**Optional**] I don't think you are making use of this class since it's already defined inside the library.js file. The probability of this code being used again is almost zero.